src/Microsoft.AspNet.Server.Kestrel/KestrelServerInformation.cs

```diff
@@ -39,5 +39,34 @@ private static ICollection<string> GetAddresses(IConfiguration configuration)
 
             return addresses;
         }
+
+        private static int GetThreadCount(IConfiguration configuration)
+        {
+            var threadCountString = configuration["server.threadCount"];
```

Comment:
@Tratcher What are your thoughts on this? Should all the properties on IKestrelServerInformation be configurable through IConfiguration? Or should Kestrel-specific things like ThreadCount and NoDelay should those only be configurable by directly modifying the IKestrelServerInformation object?